src/components/information/Information.js

```diff
@@ -1,6 +1,32 @@
 import Navigation from '../layout/Navigation'
+import { useState } from 'react'
 
 function Information() {
+	const [name, setName] = useState('')
+	const [location, setLocation] = useState('')
+	const [email, setEmail] = useState('')
+	const [bio, setBio] = useState('')
+
+	function saveNameToLocalStorage(event) {
+		setName(event.target.value)
+		window.localStorage.setItem('fullName', event.target.value)
+	}
+
+	function saveLocationToLocalStorage(event) {
+		setLocation(event.target.value)
+		window.localStorage.setItem('location', event.target.value)
+	}
+
+	function saveEmailToLocalStorage(event) {
+		setEmail(event.target.value)
+		window.localStorage.setItem('email', event.target.value)
+	}
+
+	function saveBioToLocalStorage(event) {
+		setBio(event.target.value)
+		window.localStorage.setItem('bio', event.target.value)
+	}
```

Comment:
Here you can write a function like: 

```
function saveElementToLocalStorage(event, elementName) {
    window.localStorage.setItem(elementName, event.target.value);
    switch(eventName) { 
        case 'email':
            setEmail(event.target.value); // you can also create another function for this switch 
            break;
    }
}
```

And onChange you call the function like: saveElementToLocalStorage(event, 'hereYouDecide between email, location etc'

Reply:
@Chris-Mugumwa If you need help doing it, just tell me, I can help you

Reply:
I refactored them ⛱️